frontend/src/app/side-menu/side-menu.component.ts

```diff
@@ -148,7 +128,7 @@ export class SideMenuComponent {
   uploadFiles() {
     const fileUpload = this.fileUpload.nativeElement;
     if (this.checkBox.checked) {
-      this.dashboard.newTab();
+      this.dashboard.newTab('New Tab');
```

Comment:
Is this user editable at all, or will it always just say "New Tab"?

Reply:
I was going to do a general UI update fix to address that plus a few other things. I don't think I have time to make them user editable so they'll just say something like 'TMD2725 Ambient Light'